lib/puppet_x/relay/util/http/client.rb

```diff
@@ -21,7 +25,7 @@ def request(verb, path, body: nil)
 
             update_request!(req)
 
-            http = Net::HTTP.new(uri.host, uri.port)
+            http = Net::HTTP.new(uri.host, uri.port, @proxy_host, @proxy_port, @proxy_user, @proxy_password)
```

Comment:
Should the `p_addr` arg default to `:ENV` if `@proxy_host` is nil here to retain current behavior?

Reply:
@impl Added.